src/expr/node.h

```diff
@@ -923,24 +923,100 @@ inline std::ostream& operator<<(std::ostream& out, TNode n) {
   return out;
 }
 
+namespace {
+
+template <typename T>
+void nodeContainerToOut(std::ostream& out, const T& container)
+{
+  out << "[";
+  bool is_first = true;
+  for (auto item : container)
+  {
+    out << (!is_first ? ", " : "") << item;
+    is_first = false;
+  }
+  out << "]";
+}
+
+}
+
 /**
- * Serializes a vector of node to the given stream.
+ * Serialize a vector of nodes to given stream.
  *
  * @param out the output stream to use
- * @param ns the vector of nodes to output to the stream
+ * @param the node vector to output to the stream
```

Comment:
This should probably be `@param container` to show up correctly in Doxygen.

Reply:
Done